### PR TITLE
install: support when a hyphen is passed

### PR DESCRIPTION
**Pull request by sylvestre**
Should fix: tests/install/strip-program.sh

### CODE REVIEW

File: tests/by-util/test_install.rs [hunk 1]
```diff
@@ -674,6 +674,34 @@ fn test_install_and_strip_with_program() {
     assert!(!stdout.contains(STRIP_SOURCE_FILE_SYMBOL));
 }
 
+#[cfg(all(unix, feature = "chmod"))]
+#[test]
+// FixME: Freebsd fails on 'No such file or directory'
+#[cfg(not(any(windows, target_os = "freebsd")))]
```
Comment (cakebaker) [inline tests/by-util/test_install.rs:680]: The `windows` seems unnecessary as we already have `all(unix, ...)` on line 677:
```suggestion
#[cfg(not(target_os = "freebsd"))]
```

File: src/uu/install/src/install.rs [hunk 2]
```diff
@@ -749,7 +749,21 @@ fn copy_file(from: &Path, to: &Path) -> UResult<()> {
 /// Returns an empty Result or an error in case of failure.
 ///
 fn strip_file(to: &Path, b: &Behavior) -> UResult<()> {
-    match process::Command::new(&b.strip_program).arg(to).output() {
+    // Check if the filename starts with a hyphen and adjust it
```
Comment (cakebaker) [inline src/uu/install/src/install.rs:752]:
```suggestion
    // Check if the filename starts with a hyphen and adjust the path
```

**Review by cakebaker — APPROVED** [review]
Did you notice that there is no output when running `cargo run --features="unix" install -s --strip-program ./no-hyphen -- source -dest` whereas GNU outputs `./-dest`? Should I open a ticket for it?

**Comment by sylvestre** [comment]
yeah, probably :)

**Pull request merged**